packages/core/README.md

```diff
@@ -32,7 +36,7 @@ const openai = new OpenAI({
 
 export const runtime = 'edge';
 
-export async function POST(req) {
+export default async function POST(req) {
```

Comment:
With App Router you use named exports, not defaults

Reply:
ill fix that now